src/sentry/static/sentry/app/views/performance/transactionVitals/vitalCard.tsx

```diff
@@ -311,16 +317,26 @@ class VitalCard extends React.Component<Props, State> {
                 />
               </PercentContainer>
             </Feature>
-            <BarChart
-              series={allSeries}
-              xAxis={xAxis}
-              yAxis={yAxis}
-              colors={colors}
-              onRendered={this.handleRendered}
-              grid={{left: space(3), right: space(3), top: space(3), bottom: space(1.5)}}
-              stacked
-              {...zoomRenderProps}
-            />
+            {getDynamicText({
+              value: (
+                <BarChart
+                  series={allSeries}
+                  xAxis={xAxis}
+                  yAxis={yAxis}
+                  colors={colors}
+                  onRendered={this.handleRendered}
+                  grid={{
+                    left: space(3),
+                    right: space(3),
+                    top: space(3),
+                    bottom: space(1.5),
+                  }}
+                  stacked
+                  {...zoomRenderProps}
+                />
+              ),
+              fixed: 'Web Vitals Histogram',
```

Comment:
```suggestion
              fixed: <Placeholder testId="skeleton-ui" height="200px" />
```
I've been using placeholder blocks for charts elsewhere so snapshots are accurate for page layout.